ChallengeUI/src/main/java/com/drivequant/drivekit/challenge/ui/challengedetail/viewmodel/ChallengeDetailViewModel.kt

```diff
@@ -105,6 +106,9 @@ class ChallengeDetailViewModel(private val challengeId: String) : ViewModel() {
                         val maxDuration = (it.challengeStats.maxScore * 3600 / 60).roundToInt()
                         numeratorDuration / (maxDuration - minDuration)
                     }
+                    401 -> ((it.driverStats.score - it.challengeStats.minScore) * 100).div(
```

Comment:
On pourrait peut être faire `in 101..301, in 306..309, 401` pour éviter la duplication de code

Reply:
Oui, j'ai apporté la modif, je ne voulais pas modifier trop de code mais on peut le faire là. Merci